src/drivers/neon.ts

```diff
@@ -0,0 +1,99 @@
+import { createError, createRequiredError, defineDriver } from "./utils";
+import { neon, NeonQueryFunction, neonConfig } from "@neondatabase/serverless";
+
+export interface NeonDriverOptions {
+  /**
+   * The Neon Postgres database connection string.
+   * @example "postgresql://user:password@host:port/database?sslmode=require"
+   */
+  url?: string;
+  /**
+   * The name of the table to use for storing data.
+   * @default "unstorage"
+   * @example "unstorage"
+   */
+  table?: string;
+  /**
+   * Whether to fetch connection cache or not.
+   * @default false
+   */
+  fetchConnectionCache?: boolean;
+}
+
+const DRIVER_NAME = "neon";
+
+export default defineDriver((opts: NeonDriverOptions = {}) => {
+  opts.table = opts.table || "unstorage";
+  neonConfig.fetchConnectionCache = opts.fetchConnectionCache || false;
+
+  let _connection: NeonQueryFunction<false, false>;
+
+  const getConnection = () => {
+    if (!_connection) {
+      if (!opts.url) {
```

Comment:
Would it make more sense to check this immediately, e.g on line 26 above? Fail fast(er)